internal/build/remote_config.go

```diff
@@ -0,0 +1,9 @@
+package build
```

Comment:
Thanks for moving to `pkg`. I think `build` isn't a good enough name here. Maybe `remoteconfig` ?

Reply:
build is fine because it's about vars to be replaced during build/linking.